### DISCUSSION TITLE
Name Alternative to Bytes

### DISCUSSION THREAD

**Discussion by Denneisk** [wiremod/wire#2636] [Polls]
The term "bytes" in Wiremod is often misused to refer to a Lua number, which is a [double value](https://wiki.facepunch.com/gmod/number). This is extremely misleading and could cause lots of confusion and waste in user contraptions. True byte values can still be displayed after the name change, but they should be accurate, which, as stated before, means they must be a multiple of 8.

Feel free to suggest any other alternatives and use the "Other" option. (Yes, fear not, this is the last poll update. Maximum is 8 options and we have a decent selection as it stands)

**Comment by thegrb93** [comment]
Add Byte to the poll

**Reply by Denneisk** [reply]
but... that defeats the purpose of the poll.

**Reply by thegrb93** [reply]
Maybe some people like 'byte'

**Reply by Denneisk** [reply]
"Byte" breaks the idiosyncracy that I am trying to establish here!

**Comment by vurvdev** [comment]
Could you give examples where bytes are used in wiremod?

**Reply by Denneisk** [reply]
Old memory devices such as RAM, ROM, EEPROM, CPU (and related), CD (although this one might be accurate?).
DHDD itself uses the term "values" so that's not as bad but that might be changed to the result of the poll for consistency.

**Comment by Grocel** [comment]
"Wait why does this CD have 500 kW, is there a nuclear battery inside?"

I am for C and kC.
